RIOT-OS/RIOT
# boards/stm32l476g-disco: add pinout to documentation page

### Contribution description

This PR adds board pinout to the `stm32l476g-disco` board.

### Testing procedure

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__stm32l476g-disco.html
```

### Issues/PRs references

None
